lib/timeline/Timeline.js

```diff
@@ -489,7 +505,6 @@ Timeline.prototype.getItemRange = function () {
         endSide = end   + (item.getWidthRight() + 10) * factor;
       }
 
-
       if (startSide < min) {
         min = startSide;
         minItem = item;
```

Comment:
Agree. The sign is because I'm subtracting with right while adding with left: 
```
if (this.options.rtl) {
        startSide = start - rhs * factor;
        endSide = end + lhs * factor;
      } else {
        startSide = start - lhs * factor;
        endSide = end + rhs * factor;
      }
```